### DISCUSSION TITLE
I submitted a bug report and would like to know if there is any way to immediately solve the issue rather than waiting for it to be fixed

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#16156] [Q&A]
The bug involves [placing image content in a Datagrid, causing issues with the scroll bar](https://github.com/AvaloniaUI/Avalonia/issues/16121). 

I suspect that the problem may be due to incorrect height calculations when images are placed within the Datagrid. Is it possible for me to locate and modify the source code myself to fix this issue? However, this would involve modifying the Avalonia source code and applying it to my own project, so I am unsure if it is feasible.

**Comment by timunie** [comment]
Priority support needs a support agreement. Then you can request that. Otherwise you may consider defining a height of the image

**Reply by IronEaterBeast** [reply]
Of course, I tried to define the height of the image, but it didn't work.

**Comment by stevemonaco** [comment]
It depends on the root cause. XAML frameworks are quite flexible, but if the logic is buried somewhere in a private method, then that will require a fix to Avalonia outside of copying the entire `DataGrid` codebase into your project. Avalonia does accept PRs for things like this.

You might try setting `<Image ClipToBounds="True" VerticalAlignment="Top" HorizontalAlignment="Left" ... />` and see if that helps. Often, controls larger than their parent bounds are centered, so the positioning might help. You might also explicitly set the `Image.Height` to be the same as the row height.

I looked through your issue + repro and it's not high quality. Lots of things can affect the layout engine and a column+cell template itself isn't what a contributor wants to start with. That is, they have to rebuild the rest of the app, find images (that hopefully exhibit the same characteristics as yours), and hope they can repro else they've quickly wasted 15 minutes or more when they could have been debugging instead. Maybe it's already fixed because 11.0.5 was released 8 months ago. Including a small repro that can be downloaded/cloned and immediately ran goes a long way. So would including a GIF of the behavior. Basically, make it easier for whoever will ultimately be doing the work and there's a higher chance that it actually gets fixed.

**Reply by IronEaterBeast** [reply]
Thank you for your thoughtful response. I understand and I will think of other solutions.
